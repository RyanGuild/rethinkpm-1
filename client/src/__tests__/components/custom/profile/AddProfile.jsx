import React from 'react';
import {render} from '@testing-library/react';
import AddProfile from '../../../../components/custom/profile/AddProfile';

test('render and snapshot', () => {
  let {container} = render(<AddProfile />);
  let inputs = container.querySelectorAll('input');
  expect(inputs.length).toEqual(7);
  expect(container).toMatchSnapshot();
});
