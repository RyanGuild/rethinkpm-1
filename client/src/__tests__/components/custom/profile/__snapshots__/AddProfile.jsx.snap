// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render and snapshot 1`] = `
<div>
  <h1
    class="large text-primary"
  >
    Add Profile
  </h1>
  <p
    class="lead"
  >
    <i
      class="fas fa-code-branch"
    />
  </p>
  <small>
    * = required field
  </small>
  <form
    class="form"
  >
    <div
      class="form-group"
    >
      <label>
        Address
      </label>
      <input
        name="school"
        placeholder=""
        required=""
        type="text"
        value=""
      />
    </div>
    <div
      class="form-group"
    >
      <label>
        City
      </label>
      <input
        name="degree"
        placeholder=""
        required=""
        type="text"
        value=""
      />
    </div>
    <div
      class="form-group"
    >
      <label>
        State
      </label>
      <input
        name="fieldofstudy"
        placeholder=""
        type="text"
        value=""
      />
    </div>
    <div
      class="form-group"
    >
      <label>
        Zip
      </label>
      <input
        name="fieldofstudy"
        placeholder=""
        type="text"
        value=""
      />
    </div>
    <div
      class="form-group"
    >
      <label>
        Number of Properties Owned or Managed
      </label>
      <input
        name="fieldofstudy"
        placeholder=""
        type="text"
        value=""
      />
    </div>
    <div
      class="form-group"
    >
      <label>
        Number of Total Units Owned or Managed
      </label>
      <input
        name="fieldofstudy"
        placeholder=""
        type="text"
        value=""
      />
    </div>
    <input
      class="btn btn-primary my-1"
      type="submit"
      value="Submit Profile & Add Property Information"
    />
  </form>
</div>
`;
